### DISCUSSION TITLE
Why the Gargantua dont shoots flames ?

### DISCUSSION THREAD

**Discussion by TheCelestial2000** [VJ-HLR-Developers/Half-Life-Resurgence#90] [Q&A]
There is a command to enable ?

**Comment by Cpt-Hazama** [comment]
Can you be more specific? Gargantua shoots flames just fine for us

**Reply by TheCelestial2000** [reply]
Basically, The Gargantua only attacks with stomp and melee, he simply dont do the flametower attack.
How can i solve this problem ?

**Reply by TheCelestial2000** [reply]
> Can you be more specific? Gargantua shoots flames just fine for us

When i spawn him, he do the flame sound but dont shoot, instead he do the meleee attacks, idk why everything is in order and vj base and hlr is updated

**Comment by TheCelestial2000** [comment]
> Can you be more specific? Gargantua shoots flames just fine for us

Please help me

**Comment by Cpt-Hazama** [comment]
Mkay
